src/docfx/Errors.cs

```diff
@@ -367,6 +367,12 @@ public static Error SourcePathConflict(SourceInfo<string> source)
         /// Behavior: ✔️ Message: ✔️
         public static Error RedirectionPathSyntaxError(SourceInfo<string> source)
             => new(ErrorLevel.Warning, "redirection-path-syntax-error", $"Redirection path syntax is incorrect. 'source_path' should start without '/' and 'source_path_from_root' should start with '/'.", source);
+
+        /// <summary>
+        /// Check whether there are any dangling redirection files in a repository.
+        /// </summary>
+        public static Error DanglingRedirectionFiles(IEnumerable<string> danglingRedirectionFiles)
```

Comment:
Let's make this as `ErrorLevel.Info` to avoid impact to end users.

Reply:
done